scalableminds/webknossos
# Prevent data corruption before saving

Review with "ignore whitespace".

### URL of deployed dev instance (used for testing):
- https://preventcorruption.webknossos.xyz

### Steps to test:
- open an annotation
- brush a bit, move around -> everything should work as expected
- create a new skeleton --> a hard error should appear immediately. no further saving should happen
- check the console for details. should look like this:

![image](https://github.com/user-attachments/assets/2c654ca6-c9af-42dd-a0b6-80425f088137)

### TODOs:
- [X] remove the provoking code again

### Issues:
- [Discussion](https://scm.slack.com/archives/C02H5T8Q08P/p1719418766908699)
